examples/oft-solana/tasks/solana/initConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { Keypair, PublicKey } from '@solana/web3.js'
+import { ConfigurableTaskDefinition } from 'hardhat/types'
+
+import { inheritTask } from '@layerzerolabs/devtools-evm-hardhat'
+import { type LogLevel } from '@layerzerolabs/io-devtools'
+import { type OAppConfigurator } from '@layerzerolabs/ua-devtools'
+import { TASK_LZ_OAPP_WIRE } from '@layerzerolabs/ua-devtools-evm-hardhat'
+import { initOFTAccounts } from '@layerzerolabs/ua-devtools-solana'
+
+// We'll create clones of the wire task and only override the configurator argument
+const wireLikeTask = inheritTask(TASK_LZ_OAPP_WIRE)
+
+/**
+ * Additional CLI arguments for our custom wire task
+ */
+interface Args {
+    logLevel: LogLevel
+    solanaProgramId: PublicKey
+    solanaSecretKey?: Keypair
+    multisigKey?: PublicKey
+    internalConfigurator?: OAppConfigurator
+}
+
+// This task will use the `initOFTAccounts` configurator that initializes the Solana accounts
+const initConfigTask = wireLikeTask('lz:oft:solana:init-config') as ConfigurableTaskDefinition
```

Comment:
Shouldn't README.md be updated for this too?

Reply:
you're right. updating now..

Reply:
@ryandgoulding pushed